src/test/java/XmlParserTest.java

```diff
@@ -0,0 +1,20 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class XmlParserTest {
+
+    @Test
+    void should_should_return_users_from_users_xml() {
+        // given
+        String file = "users.xml";
+
+        // when
+        var result = XmlParser.parser(file);
+
+        // then
+        assertEquals(result,"name: Kalle Anka email: donald@email.dt userName: donaldd\n" +
```

Comment:
When you split your class to 3 smaller ones, only parsing class should be tested, it should ease testing